core/src/test/java/org/axonframework/commandhandling/model/AggregateDescriptorSerializationTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.commandhandling.model;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.axonframework.serialization.SerializedObject;
+import org.axonframework.serialization.xml.XStreamSerializer;
+import org.junit.*;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+import static org.junit.Assert.*;
+
+public class AggregateDescriptorSerializationTest {
```

Comment:
A short javadoc about what is tested here wouldn't hurt ;)
Btw, I like this ser/deser test.

Reply:
Yea good one, this might be a bit to weird on face value.
And thanks! :)